scripts/setupEnvVars.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 
-DAISYHAT_DOCKER_IMAGE="actions-image" # name of the daisyhat docker image
-DAISYHAT_CONTAINER_NAME="daisyhat-container" # desired name of the docker container
-DAISYHAT_RUNNER_NAME="daisyhat-runner" # desired name of the github actions runner
+export DAISYHAT_DOCKER_IMAGE="actions-image" # name of the daisyhat docker image
+export DAISYHAT_CONTAINER_NAME="daisyhat-container" # desired name of the docker container
+export DAISYHAT_RUNNER_NAME="daisyhat-runner" # desired name of the github actions runner
 
 # Uncomment these or add new lines below:
-# DAISYHAT_DEVICE_MAJOR=189 # major device number of devices to passthrough to the container
-# DAISYHAT_REPO="daisyhat_Examples" # the github repo to connect to
-# DAISYHAT_USER="TheSlowGrowth" # the github username
-# DAISYHAT_GH_TOKEN="" # the github access token that allows to add/remove runners
\ No newline at end of file
+# export DAISYHAT_DEVICE_MAJOR=189 # major device number of devices to passthrough to the container
+# export DAISYHAT_REPO="daisyhat_Examples" # the github repo to connect to
+# export DAISYHAT_USER="TheSlowGrowth" # the github username
+# export DAISYHAT_GH_TOKEN="" # the github access token that allows to add/remove runners
+
```

Comment:
The `Appended from ...` comment that you had here in one of the comits could actually be added from the setup script. I like it as a reminder to understand what's going on when you look at this file long after the initial setup.

Reply:
`raspberryPiSetup.sh` does append this. It was just still in there after a test run I did.  Those commits were just to clean up.

Reply:
Ah, my bad :-)